frontend/src/components/ResourceActionDialog.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <v-dialog v-model="isOpen" id="resource-action-dialog" width="500">
+    <v-card class="pa-6">
+      <div class="mt-n6 mx-n6 mb-4 pa-4 d-flex" style="background-color: #F5F5F5">
+        <v-spacer></v-spacer>
+        <v-btn outlined color="primary" @click="$emit('close')">
+          Fermer
+        </v-btn>
+      </div>
+
+      <h2 id="modal-title" class="mb-3" tabindex="-1">Mis en place ?</h2>
```

Comment:
```suggestion
      <h2 class="mb-3" tabindex="-1">Mis en place ?</h2>
```
pourquoi le tabindex ?